### PR TITLE
Update members

### PR DESCRIPTION
**Pull request by hzoo**
Not sure how we want to make the changes so just an idea?

cc @loganfsmyth @danez. @kittens

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,12 @@ We've started the process of bringing together various communities using this fo
 * Ingvar Stepanyan ([Acorn](https://github.com/ternjs/acorn))
 * Mike Sherov ([Esprima](https://github.com/jquery/esprima))
 * Michael Ficarra ([@michaelficarra](https://github.com/michaelficarra))
-* Sebastian McKenzie ([Babel](https://github.com/babel/babel))
 * Kyle Simpson ([@getify](https://github.com/getify))
+* Henry Zhu, Logan Smyth, Daniel Tschinder ([Babel](https://github.com/babel))
```
Comment (hzoo) [inline README.md]: could make these links?

**Comment by mikesherov** [comment]
👍  @RReverser want to squash and merge?

**Comment by RReverser** [comment]
@mikesherov You can do that too :)

**Comment by hzoo** [comment]
Or I can do it when I can merge 😄

**Comment by RReverser** [comment]
@hzoo One question though - it appears that you're the only one actively participating in issues here, and for other projects we de-facto included one representative per each, not every project maintainer. I'm not trying to diminish work of @loganfsmyth or @danez in any way (!), they're doing great job on Babel, just wanted to ask if that's intentional and you're all going to be participants in ESTree or maybe want to have one representative.

**Comment by mikesherov** [comment]
@RReverser I just meant that i approve, and whn you approve, merge away... Didn't mean that you should do the work instead of me. And look... you had feedback!

**Comment by RReverser** [comment]
@mikesherov Yeah, I understood, just meant you don't really need my approval for these kinds of things I think :)

**Comment by mikesherov** [comment]
I think it's fine leaving all three. We can always update later.

**Comment by hzoo** [comment]
I believe I would be pinging them on the issues anyway, but yeah I or them could be a single "rep". They've been more involved with Babylon in general although I just try to figure out the coordination of the project in general - either works

**Comment by nzakas** [comment]
I've been involved from the start, no love? :(

**Comment by hzoo** [comment]
sounds good to me @nzakas, I think the list was out of date in general

**Comment by mikesherov** [comment]
@nzakas, I love you despite what the README says! But yeah let's add you.

**Comment by nzakas** [comment]
:heart:

**Pull request merged**